deploy/cloudformation/static-host.yml

```diff
@@ -139,10 +134,8 @@ Resources:
             - OPTIONS
           Compress: true
           DefaultTTL: !FindInMap [CacheSettings, !Ref EnvType, "DefaultTTL"]
-          ViewerProtocolPolicy: !If
-            - NoSSL
-            - allow-all
-            - redirect-to-https
+          ViewerProtocolPolicy: 
+            redirect-to-https
```

Comment:
Since this is now just a literal value and not an array of params, can put this on the same line as the ViewerProtocolPolicy

Reply:
Fixed in latest version